docs/_reference/events-api.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "@slack/events-api"
+slug: events-api
+---
+
+<h1 id="slackevents-api">@slack/events-api</h1>
+<h2 id="functions">Functions</h2>
+<h3 id="createeventadaptersigningsecret-options">createEventAdapter(signingSecret, options)</h3>
+<p>Creates a new <code>SlackEventAdapter</code>.</p>
+<strong>Parameters:</strong>
+<table>
+<thead>
+<tr>
+<th align="center">Name</th>
+<th align="center">Type</th>
+<th align="center">Required</th>
+<th></th>
```

Comment:
Hmm, curious why all these parameter tables are generating an extra column 🤔... Is `filterUndef` not working, or...?